src/utils/proxy-console.js

```diff
@@ -200,6 +200,12 @@
       }
 
       const replacements = args[0].match(/(%[sc])([^%]*)/gm)
+
+      // If no replacement is found, we return a unstyled text
+      if (!replacements || replacements === null) {
```

Comment:
`if (!replacements) {`

is the same but shorter

Reply:
actually pleace change to

`if (!repalcements || !replacements.length) {`

because we do replacements.shift() below that

Reply:
just did the change!